DependencyInjection/Configuration.php

```diff
@@ -388,6 +388,7 @@ private function addExceptionSection(ArrayNodeDefinition $rootNode)
                             ->useAttributeAsKey('name')
                             ->prototype('boolean')->end()
                         ->end()
+                        ->booleanNode('debug')->treatNullLike(false)->end()
```

Comment:
Boolean node does not allow to use other value that boolean. Thats why i was injecting 'kernel.debug' value previously.
Without `treatNullLike(false)` _null_ will be treated as _true_